zcash_client_backend/src/keys.rs

```diff
@@ -606,8 +640,11 @@ impl UnifiedFullViewingKey {
     /// Returns the Unified Address corresponding to the smallest valid diversifier index,
     /// along with that index.
     pub fn default_address(&self) -> (UnifiedAddress, DiversifierIndex) {
-        self.find_address(DiversifierIndex::new())
-            .expect("UFVK should have at least one valid diversifier")
+        self.find_address(
+            DiversifierIndex::new(),
+            &[Typecode::P2pkh, Typecode::Sapling],
```

Comment:
TODO (maybe not in this PR): conditionally add Orchard here.

Reply:
Since we now have the `orchard` feature in `main`, I think it makes sense in this PR.